bin/legacy/commands/rerun-workflow.js

```diff
@@ -0,0 +1,6 @@
+const { builder, handler } = require('../../cml/workflow/restart');
```

Comment:
delete this file. Not needed as per https://github.com/iterative/cml/pull/1073#pullrequestreview-1017307920

Reply:
Absolutely needed as poer https://github.com/iterative/terraform-provider-iterative/blob/823ab139473df13bd0a9fb2adb472941d27b8e27/task/common/machine/machine-script.sh.tpl#L15